app/src/main/res/xml/appfilter.xml

```diff
@@ -2253,21 +2345,24 @@
     <item component="ComponentInfo{im.vector.alpha/im.vector.activity.LoginActivity}" drawable="riot" />
     <item component="ComponentInfo{im.vector.alpha/im.vector.activity.SplashActivity}" drawable="riot" />
 
-    <!-- RiotX -->
-    <item component="ComponentInfo{im.vector.riotx/im.vector.riotx.features.Alias}" drawable="riot" />
```

Comment:
I would move this to the Element block and use the new icon just so people still using the old app won't have a non-frost icon with the next update.

Reply:
yes fixed this one